version.properties

```diff
@@ -1 +1 @@
-version=2.2.0
+version=2.2.1
```

Comment:
This is a breaking change to the API, no? We've removed options from tasks, etc. Perhaps we bump to 3.0.0?